optuna/visualization/_parallel_coordinate.py

```diff
@@ -97,6 +98,7 @@ def _get_parallel_coordinate_plot(
 ) -> "go.Figure":
 
     layout = go.Layout(title="Parallel Coordinate Plot")
+    reverse_scale = _is_reverse_scale(target, study.direction)
```

Comment:
If `target` is `None`, `target` is replaced with a method `_target ` at L122, so we need to store `reverse_scale` here.